rapidsai/devel-rocky.arm64.Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG FROM_IMAGE=gpuci/miniforge-cuda-arm64
-ARG CUDA_VER=11.2
-ARG LINUX_VER=centos8
+ARG FROM_IMAGE=gpuci/miniforge-cuda
```

Comment:
Is there a reason you changed this `FROM_IMAGE`? It shouldn't need to be changed.

Reply:
Fixed.  No reason this change was a correction after the FROM_IMAGE was changed back after the original switch from nvidia/cuda.